src/components/common/Extensions/ImageCarousel/consts.tsx

```diff
@@ -1,5 +1,5 @@
 export const images = [
-  'https://res.cloudinary.com/ifeomaimoh/image/upload/v1652345767/demo_image2.jpg',
-  'https://res.cloudinary.com/ifeomaimoh/image/upload/v1652345874/demo_image1.jpg',
-  'https://img.freepik.com/free-vector/hand-painted-watercolor-pastel-sky-background_23-2148902771.jpg?w=2000',
+  'https://source.unsplash.com/450x300/?oahu?r=1',
```

Comment:
why that change? PR related to Tabs component not carousel

Reply:
Changes are reverted